### DISCUSSION TITLE
切换子应用的不同路由过程中，主应用的container组件也会卸载吗

### DISCUSSION THREAD

**Discussion by gitshuang** [umijs/qiankun#2890] [Q&A]
在做vue2 多tab主应用，从主应用菜单中有两个子应用路由，点击路由1的时候，能够加载子应用相应路由1页面，且主应用tab页签上追加了一个路由1 的tab，点击路由2的时候，页面url地址栏一闪而过路由2的url，然后跳转到主应用默认路由页面，主应用tab页签上也追加了一个路由2的tab.，这时候所有的tab都不能点击了。

本地开发没有问题，发布线上就出现上述问题，看控制台，发现本地开发切换tab时候不触发container子应用容器组件卸载，也不触发子应用卸载。但是线上先触发了container卸载，又触发了自用应用卸载，但是tab功能瘫痪

**Comment by gitshuang** [comment]
只打开一个子应用的路由tab，再关闭，会报这个错Wrapper element for microappauth is not existed!

**Comment by SeanLiangYoung** [comment]
遇到了同样的问题,蹲一个解决方案
